.github/workflows/check-rtw.yml

```
@@ -28,6 +28,7 @@ jobs:
   check-rtw:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
+    if: github.repository == 'ESMValGroup/ESMValTool'  # avoid GAs in forks
```

This workflow is run both on push and schedule. Would there be some way to still run it on pushes, but not on schedule?

---

Yes, maybe not important though, as long as stuff runs on pull requests, but were not using ghas for those

---

afraid I don't quite understand what you mean, bud - should I just let this one free to run anywhere (ie on forks too?)?

---

What I mean is that the workflow should be run on `push` events even in forks (but not on `schedule` events), how else can people see that they got it right when making a pull request from forks?


[This](https://stackoverflow.com/questions/59349905/which-properties-does-github-event-in-a-github-workflow-have) post seems to suggest that something like this might work:
```suggestion
    if: github.event_name != 'schedule' || github.repository == 'ESMValGroup/ESMValTool' # avoid scheduled GHAs in forks
```

---

Thanks! They are useful to run on forks if you're using a fork to develop these things in. Could you change the citation file check too please?

---

on it now, buds

---

le done in le commite https://github.com/ESMValGroup/ESMValTool/pull/3894/commits/e38637146f40775145ce8c2da31386ec070f15e0